game-core/src/main/java/games/strategy/engine/framework/startup/ui/posted/game/pbf/test/post/TestPostAction.java

```diff
@@ -53,7 +54,12 @@ public void accept(final String forumName, final Integer topicId) {
                           + ", time: "
                           + DateTimeUtil.getLocalizedTime(),
                       "Testing Forum poster",
-                      f != null ? f.toPath() : null);
+                      f != null
+                          ? SaveGameParameter.builder()
+                              .path(f.toPath())
+                              .displayName("Test.jpg")
```

Comment:
Fun Fact:
I changed it back to the original name: https://github.com/triplea-game/triplea/blame/a0d65198458522e08abc9fba36543a1494cb2058/game-core/src/main/java/games/strategy/engine/framework/startup/ui/editors/ForumPosterEditor.java#L139